### DISCUSSION TITLE
Deepfillv2 test result different from doc.

### DISCUSSION THREAD

**Discussion by weisy11** [open-mmlab/mmediting#1218] [Models]
I used the command:
`python tools/test.py configs/inpainting/deepfillv2/deepfillv2_256x256_8x2_places.py models/deepfillv2_256x256_8x2_places_20200619-10d15793.pth`
and got the result:
Eval-l1: 0.18419936701791897
Eval-psnr: 21.784859316873575
Eval-ssim: 0.8054377584437133

psnr and ssim slightly different from the [doc](https://github.com/open-mmlab/mmediting/blob/master/configs/inpainting/deepfillv2/README.md)，and l1 seems quite different.
Which Places365-val do you use? Standard or 256?
Many thx.

**Comment by zengyh1900** [comment]
@Z-Fran please check this.
